src/components/ListItem.jsx

```diff
@@ -36,6 +37,18 @@ export function ListItem({ name, items, listToken }) {
 		}
 	};
 
+	const handleDeleteItem = () => {
+		const confirm = window.confirm(`Do you really want to delete ${name}`);
+
+		try {
+			if (confirm) {
+				deleteItem(listToken, id);
+			}
+		} catch (error) {
+			console.log(error);
+		}
+	};
```

Comment:
If you want to catch errors coming from `deleteItem` you should make the `handleDeleteItem` function async and await the result to `deleteItem` like this:
```
	const handleDeleteItem = async () => {
		const confirm = window.confirm(`Do you really want to delete ${name}`);

		try {
			if (confirm) {
				await deleteItem(listToken, id);
			}
		} catch (error) {
			console.log(error);
		}
	};
```

Reply:
Just reworked this, thank you! :)